packages/core/src/options.ts

```diff
@@ -112,10 +112,7 @@ export function optionsWithDefaults(options?: Options): Required<Options> {
     staticHelpers: false,
     staticModifiers: false,
     staticComponents: false,
-    packageRules: [],
```

Comment:
Not 100% sure about this one, but it isn't defined in the interface above and it's also [a compat option](https://github.com/embroider-build/embroider/blob/main/packages/compat/src/options.ts#L85).